crates/sparrow-main/tests/e2e/basic_error_tests.rs

```diff
@@ -1086,3 +1088,32 @@ async fn test_unsupported_output_type_with_nested_formulas_2() {
           - ""
     "###);
 }
+
+#[tokio::test]
+async fn test_swapped_args_for_get_map() {
```

Comment:
Suggest putting this in `map_tests`. I think we should get rid of the `error_tests` and just test errors alongside the valid cases (the fixtures more often line up, etc.).